op-e2e/sequencer_failover_setup.go

```diff
@@ -32,10 +30,11 @@ import (
 )
 
 const (
-	sequencer1Name = "sequencer1"
-	sequencer2Name = "sequencer2"
-	sequencer3Name = "sequencer3"
-	verifierName   = "verifier"
+	Sequencer1Name = "sequencer1"
+	Sequencer2Name = "sequencer2"
+	Sequencer3Name = "sequencer3"
+	Sequencer4Name = "sequencer4"
+	VerifierName   = "verifier"
```

Comment:
I notice Sequencer4 is mentioned here but never used, consider removing?

Reply:
It will be used in later tests (for deployment situations), I think we can keep it here since no complaints from CI.